packages/api-reference/src/components/Content/Content.vue

```diff
@@ -91,7 +91,8 @@ const introCardsSlot = computed(() =>
 
     <Introduction
       v-if="document?.info?.title || document?.info?.description"
-      :document="document">
+      :document="document"
+      :externalDocs="activeCollection?.externalDocs">
```

Comment:
why do we need the activeCollection here? can't we just use document.info.externalDocs?

Reply:
but i guess this has changed in the meantime, will update